src/assets/js/partials/product-card.js

```diff
@@ -231,7 +231,7 @@ class ProductCard extends HTMLElement {
               : ``}
           </div>
           ${this.product?.donation && !this.minimal && !this.fullImage ?
-          `<salla-progress-bar donation=${this.product?.donation} />
+          `<salla-progress-bar donation=${JSON.stringify(this.product.donation)} /></salla-progress-bar>
```

Comment:
remove `</salla-progress-bar>`
`/>` should do the same, or remove `/` from `/>`